jena-fuseki2/jena-fuseki-webapp/src/main/java/org/apache/jena/fuseki/cmd/JettyFusekiWebapp.java

```diff
@@ -77,7 +77,7 @@
     // Standalone jar
     public static final String resourceBase1   = "webapp";
```

Comment:
Right, I think the ZIP file will call the standalone jar (when used with `java -jar fuseki-server.jar`). So this `webapp` will be used.

When using in Eclipse I had to modify the line below to load the new UI.